packages/notifications-library/src/notifications_library/payments.py

```diff
@@ -26,6 +26,7 @@ class PaymentData:
     price_dollars: str
     osparc_credits: str
     invoice_url: str
+    invoice_pdf: str
```

Comment:
if this is a link, please use a link type as well.

Reply:
done